spec/features/visitor/password_reset_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "Password Reset" do
+  let(:new_password) { "qwe123" }
+  let(:user) { create :user, :confirmed }
+
+  def update_password
```

Comment:
to make related scenario smaller and more focused on what's important: verify that user can reset his password through password reset email.